user/conditional-builds-stages-jobs.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Conditional Builds, Stages, and Jobs
+layout: en
+---
+
+## Conditional Builds, Stages, and Jobs
+
+You can filter out, and reject builds, stages, and jobs by specifying conditions in your build configuration (your `.travis.yml` file, or via [API build requests](https://docs.travis-ci.com/user/triggering-builds/)).
```

Comment:
It would be good to have an API example or two.

Reply:
After discussing on Slack we've decided to just remove this, as usecases would be rather sophisticated.